projects/core/components/dialog/dialog.component.ts

```diff
@@ -71,6 +77,12 @@ export class TuiDialogComponent<O, I> {
             : SMALL_DIALOGS_ANIMATION;
     }
 
+    @HostListener('click', ['$event.target']) hostClick(target: HTMLElement) {
```

Comment:
@MarsiBarsi shouldn't prettier put the method name on a new line?

Reply:
Unfortunately it shouldn’t. It is just our agreement 😔

Reply:
Done